raiden/transfer/channel.py

```diff
@@ -113,6 +113,18 @@
 SendUnlockAndMerkleTree = Tuple[SendBalanceProof, MerkleTreeState]
 
 
+def get_sender_expiration_threshold(lock: HashTimeLockState) -> BlockNumber:
```

Comment:
please, add a docstring explaining what this does

Reply:
This is really confusing, why not just:

> Returns the block number at which the sender can send the remove expired lock.
>
> The remove lock expired message will be rejected if the expiration block has not been confirmed,
> additionally the sender can account for possible delays in the receiver, so a few additional
> blocks are used to avoid hanging the channel.